### ISSUE TITLE
Reliance on Node Globals / Built-ins breaks in browser / build tools.

### ISSUE THREAD

**Issue by bootsa** [citation-js/citation-js#150]
Relying on Node core modules like `process` in this [line](https://github.com/citation-js/citation-js/blob/1af82342ea5b216daba0aee492c6b56f57c2c999/packages/core/src/util/fetchFile.js#L11) 
```js
let userAgent = `Citation.js/${pkg.version} Node.js/${process.version}`
```
breaks in environments where Node core modules aren't included / polyfilled - like [Sveltekit](https://kit.svelte.dev/) / [Vite](https://vitejs.dev/) and I think [Webpack](https://webpack.js.org/configuration/node) > 4, though I don't use it.

As `process.version` is used here solely for extra identification, I'd suggest removing it or at least giving it a simple fallback string.

In Sveltkit I got around it by passing an object through the build tool (Vite) - see https://github.com/InvasionBiologyHypotheses/citationJS-sveltekit-demo/blob/e8a4d99e74589d7110f0560f8ce97befe6e4569a/svelte.config.js#L9 - though it was an additional hurdle to getting Citation JS up and running.

**Comment by larsgw** [comment]
Thank you for the report. I hadn't started working on a fix yet because I wasn't sure how to check that I caught every occurrence.

**Comment by larsgw** [comment]
Fixed in d779267d8579e0e50390834b447c6b83cd446645
